tests/dragonfly/replication_test.py

```diff
@@ -1836,3 +1836,95 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     await c_replica.close()
     master.stop()
     replica.stop()
+
+
+from random import choice
+
+
+def random_str(len):
```

Comment:
I think this is redundant

Reply:
this was from previous parameter tweaking... just cleaned up